model/src/main/kotlin/utils/ResolutionProvider.kt

```diff
@@ -27,7 +27,7 @@ import org.ossreviewtoolkit.model.config.Resolutions
 import org.ossreviewtoolkit.model.config.RuleViolationResolution
 
 /**
- * A provider for resolutions of [OrtIssue]s.
+ * An interface to provide resolutions for [OrtIssue]s.
```

Comment:
This is outdated, it also provides resolutions vor `[RuleViolation]s`.

Reply:
Thanks, please see https://github.com/oss-review-toolkit/ort/pull/2655.